packages/esm-patient-search-app/src/config-schema.ts

```diff
@@ -21,6 +45,152 @@ export const configSchema = {
+        },
+        dateOfBirth: {
+          _type: Type.Object,
+          _description: 'Configuration for the date of birth field',
+          enabled: {
+            _type: Type.Boolean,
+            _description:
+              'Optional. If true, determines whether to display the date of birth field or not. Defaults to true',
+            _default: true,
+          },
+          label: {
+            _type: Type.String,
+            _description: 'The label text for date of birth field',
+            _default: 'Date of Birth',
+          },
+        },
+        age: {
+          _type: Type.Object,
+          _description: 'Configuration for the age field',
+          enabled: {
+            _type: Type.Boolean,
+            _description: 'Optional. If true, determines whether to display the age field or not. Defaults to true',
+            _default: true,
+          },
+          label: {
+            _type: Type.String,
+            _description: 'The label text for the age field',
+            _default: 'Age',
+          },
+          min: {
+            _type: Type.Number,
+            _description: 'The minimum value for the age field',
+            _default: 0,
+          },
+          max: {
+            _type: Type.Number,
+            _description: 'The maximum value for the age field',
+            _default: 0,
+          },
+        },
+        postcode: {
+          _type: Type.Object,
+          _description: 'Configuration for the postcode field',
+          enabled: {
+            _type: Type.Boolean,
+            _description:
+              'Optional. If true, determines whether to display the postcode field or not. Defaults to true',
+            _default: true,
+          },
+          label: {
+            _type: Type.String,
+            _description: 'The label text for the postcode field',
+            _default: 'Postcode',
+          },
+        },
+      },
+      personAttributes: {
+        _type: Type.Array,
+        _description: 'Configuration for person attributes to display on advanced search',
+        _elements: {
+          _type: Type.Object,
+          placeholder: {
+            _type: Type.String,
+            _description: 'Placeholder text for the field',
+            _default: '',
+          },
+          attributeTypeUuid: {
+            _type: Type.UUID,
+            _description: 'UUID of the person attribute type',
+          },
+          answerConceptSetUuid: {
+            _type: Type.ConceptUuid,
+            _default: null,
+            _description:
+              'For coded questions only. A concept which has the possible responses either as answers or as set members.',
+          },
+          customConceptAnswers: {
+            _type: Type.Array,
+            _default: [],
+            _elements: {
+              _type: Type.Object,
+              uuid: {
+                _type: Type.UUID,
+                _description: 'Answer concept UUID',
+              },
+              label: {
```

Comment:
Do not allow passing labels, it's another hassle to handle the translations, the labels should be used from the backend.